Src/Fido2.Models/CredentialCreateOptions.cs

```diff
@@ -55,6 +55,34 @@ public sealed class CredentialCreateOptions : Fido2ResponseBase
     [JsonPropertyName("authenticatorSelection")]
     public AuthenticatorSelection AuthenticatorSelection { get; set; }
 
+    private List<PublicKeyCredentialHint> _hints;
+
+    /// <summary>
+    /// Guides the user agent in interacting with the user. This OPTIONAL member contains zero or more elements from <see cref="PublicKeyCredentialHint" />.
+    /// </summary>
+    /// <remarks>
+    /// When <see cref="Hints"/> is set, <see cref="AuthenticatorSelection.AuthenticatorAttachment" /> will be set to <see cref="AuthenticatorAttachment.CrossPlatform" /> for compatibility with older user agents.
+    /// </remarks>
+    [JsonPropertyName("hints")]
+    [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
+    public List<PublicKeyCredentialHint> Hints
+    {
+        get
+        {
+            return _hints;
+        }
+        set
+        {
+            if (value != null && value.Any())
+            {
+                AuthenticatorSelection.AuthenticatorAttachment ??= new AuthenticatorAttachment();
+                AuthenticatorSelection.AuthenticatorAttachment = AuthenticatorAttachment.CrossPlatform;

```

Comment:
I don't think this is correct, see the spec:
![CleanShot 2024-07-11 at 14 06 51@2x](https://github.com/passwordless-lib/fido2-net-lib/assets/357283/5d8a4549-cb78-42f9-b678-23405d7c91dd)

Reply:
@abergs Ah right the second one has to be `platform`. I completely missed that.

Reply:
@abergs Ready.